MAUI/visual-studio-integration/Essential-UI-Kit.md

```diff
@@ -1,21 +1,21 @@
 ---
 layout: post
 title: Essential® UI Kit for .NET MAUI | MAUI | Syncfusion®
-description: The Syncfusion® Essential® UI Kit for .NET MAUI extension provides the predefined design Screens for the MAUI Apps.
-platform: maui
-control: Syncfusion<sup>®</sup> Extensions
-documentation: ug
+description: The Syncfusion® Essential® UI Kit for .NET MAUI extension provides the predefined design screens for the MAUI apps.
+platform: MAUI
+control: Syncfusion® Extensions
+documentation: UG
 ---
 
 # Essential<sup>®</sup> UI Kit for .NET MAUI
 
-Essential<sup>®</sup> UI Kit for .NET MAUI for MAUI comes with ready-to-use XAML templates, enabling you to effortlessly design user interfaces for cross-platform applications. With a clear separation of the View, View Model, and Model classes, which makes the integration of your business logic and making changes to existing views simple.
+The Essential<sup>®</sup> UI Kit for .NET MAUI offers ready-to-use XAML templates, allowing you to effortlessly design user interfaces for cross-platform applications. It facilitates a clear separation of the View, ViewModel, and Model classes, making it simple to integrate your business logic and modify existing views.
 
 ## Installation of Essential<sup>®</sup> UI Kit for .NET MAUI Extension
 
-To get started with the Essential<sup>®</sup> UI Kit for .NET MAUI, first install the appropriate extension in Visual Studio by downloading it from the marketplace. As a result, you’ll be able to access and utilize the Syncfusion<sup>®</sup> Extension directly from the Syncfusion<sup>®</sup> menu within your project
+To begin with the Essential<sup>®</sup> UI Kit for .NET MAUI, install the appropriate extension in Visual Studio by downloading it from the marketplace. As a result, you’ll be able to access and utilize the Syncfusion<sup>®</sup> Extension directly from the Syncfusion<sup>®</sup> menu within your project.
 
-[Visual Studio 2022]()
+[Visual Studio 2022](https://marketplace.visualstudio.com/items?itemName=SyncfusionInc.MAUIVSExtension)
```

Comment:
@snehakPV, replace with the below hyperlink,

https://marketplace.visualstudio.com/items?itemName=SyncfusionInc.Essential-UI-Kit-MAUI